p2panda-js/src/entry/index.ts

```diff
@@ -0,0 +1,3 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+export * from './entry';
```

Comment:
When reading this PR I found myself checking all `index.ts` files first to understand what the modules per folder export, which was really nice! Except of this case .. and the `session/index.ts` where we use a wildcard import. This is really a detail, just wanted to suggest to make exports explicit here as well for readability.

Reply:
Good point! I was also thinking about this. The downside I see is that you might `export` a new symbol from a module without updating the `index.ts` and then wonder why you can't import it elsewhere. But having an overview of what is contained in a folder is also very useful so I'll change this.

Btw: I mostly set up the `index.ts` files like this to be able to distinguish filenames without looking at their path, i.e. my editor and file search display filenames only and it's frustrating if all of the modules are `index.ts` because they all look the same. Putting the "meat" of the module in a file with a descriptive name solves this problem.